### PR TITLE
 rewrite the document of CMAQ

### PR DESCRIPTION
**Pull request by HappyCockroach**

### CODE REVIEW

File: CMAQ.md [hunk 1]
```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
 
 ### Installing dependencies
 
-```shell
+
 yum install time -y
 yum install curl* -y
-yum install wget -y
```
Comment (martin-g) [inline CMAQ.md:39]: `wget` is needed later to fetch the .tar.gz files

**Comment by Yikun** [comment]
@HappyCockroach It's better to let @martin-g take a look before force merge.

**Comment by martin-g** [comment]
I will review and test it anyway. Just testing the removal of `--enable-mpi-fortran=yes` for openmpi

File: CMAQ.md [hunk 2]
```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
 
 ### Installing dependencies
 
-```shell
+
```
Comment (martin-g) [inline CMAQ.md:35]: This broke the Markdown rendering. See https://github.com/Yikun/hpc-arm/blob/main/CMAQ.md#installing-dependencies
It should be either one `yum install -y ...` line or the formatting should be fixed to render several lines

Comment (martin-g) [reply CMAQ.md:35]: It seems the Markdown formatting for the whole file is gone now.

File: CMAQ.md [hunk 3]
```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
...
 
 2. Install Open MPI:
 
-```shell
+
 wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.3.tar.gz
 tar -zxvf openmpi-4.0.3.tar.gz
 cd openmpi-4.0.3
-./configure --prefix=/path/to/OPENMPI --enable-pretty-print-stacktrace --enable-orterun-prefix-by-default  --with-cma --enable-mpi1-compatibility --enable-mpi-fortran=yes
-make -j $(nproc) all
```
Comment (martin-g) [inline CMAQ.md:71]: The removal of `all` target leads to https://github.com/Yikun/hpc-arm/pull/1#discussion_r686779754

File: CMAQ.md [hunk 4]
```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
...
-   c. Press Esc, enter :wq!, and press Enter to save the file and exit.
 
 6. Run the following commands to copy Makefile and configure HOME:
 
-```shell
-cp ioapi/Makefile.nocpl ioapi/Makefile
-```
-
+   cp ioapi/Makefile.nocpl ioapi/Makefile
+   export HOME=/path/to/CMAQ
```
Comment (martin-g) [inline CMAQ.md:180]: Overwriting `$HOME` looks fishy

File: CMAQ.md [hunk 5]
```diff
@@ -33,313 +32,243 @@ Brief description: a three-dimensional Euler grid-based atmospheric chemistry an
...
-
-    c. Press Esc, enter :wq!, and press Enter to save the file and exit.
 
 18. Run the following command to connect to various dependent libraries:
-
-```shell
-./config_cmaq.csh gcc 9.3.0
-```
+  
+    ./config_cmaq.csh gcc 9.3.0
```
Comment (martin-g) [inline CMAQ.md:267]:
```
# ./config_cmaq.csh gcc 9.3.0
Compiler is set to gcc
0: Event not found.
```

I am not sure what this error means.

**Pull request merged**